week1_welcome/setup-windows.md

```diff
@@ -19,35 +19,49 @@ c:\Users\you>python --version
 3.10.11
 ```
 Q: What if it doesnt work?<br/>
-A: If you have newer versions of Python installed you may see a higher version of Python reported.
+A: If you have other versions of Python installed you may see a different version of Python reported.
 
 Attempt these remedies and return to step 4 above:
 - Uninstall other versions of Python
 - Remove conflicting PATH entries (dont forget to restart your command prompt before outputting the Python version again)
 
 ## Install Java 1.8
-Navigate to https://adoptium.net/download/:
+1. Navigate to the adoptium.net downloads page using [this link which will preconfigure the download filters](https://adoptium.net/temurin/releases/?os=windows&arch=x64&package=jdk&version=8). After navigating to the page you should see these settings selected:
+    - Operating System = Windows
+    - Architecture = x64
+    - Package Type = JDK
+    - Version = 8 - LTS
 
-Filter on:
-* Operating System = Windows
-* Architecture = x64
-* Package Type = JDK
-* Version = 8 - LTS
+2. Download and execute the .msi Zulu Microsoft Installer.
```

Comment:
We aren't using Zulu, we're using Eclipse Temurin?

Reply:
Addressed in next PR.